packages/contracts/src/contracts/tokens/rsk.ts

```diff
@@ -8,7 +8,14 @@ export const rsk: Record<string, string> = {
   xusd: '0xb5999795BE0EbB5bAb23144AA5FD6A02D080299F',
   dllr: '0xc1411567d2670e24d9C4DaAa7CdA95686e1250AA',
   mynt: '0x2e6B1d146064613E8f521Eb3c6e65070af964EbB',
```

Comment:
Do we get any errors in code if we remove `mynt` token and all references from contract package? MYNT token is no longer a thing, and we should remove it if possible

Reply:
@soulbit will try to test it